PROTO_tests/tests/pdarray_creation_test.py

```diff
@@ -0,0 +1,762 @@
+        assert printable_randoms == pda.to_list()
+
+        pda = ak.random_strings_lognormal(
+            np.int64(2), np.float64(0.25), np.int64(10), seed=1, characters="printable"
+        )
+        assert printable_randoms == pda.to_list()
+
+    def test_mulitdimensional_array_creation(self):
+        av = ak.array([[0, 0], [0, 1], [1, 1]])
+        assert isinstance(av, ak.ArrayView)
+
+    @pytest.mark.parametrize("size", pytest.prob_size)
+    @pytest.mark.parametrize("dtype", [bool, np.float64, np.int64, str])
+    def test_from_series_dtypes(self, size, dtype):
+        p_array = ak.from_series(pd.Series(np.random.randint(0, 10, size)), dtype)
+        assert isinstance(p_array, (ak.pdarray, ak.Strings))
+        assert dtype == p_array.dtype
+
+        p_i_objects_array = ak.from_series(
+            pd.Series(np.random.randint(0, 10, size), dtype="object"), dtype=dtype
+        )
+        assert isinstance(p_i_objects_array, (ak.pdarray, ak.Strings))
+        assert dtype == p_i_objects_array.dtype
+
+    def test_from_series_misc(self):
+        p_array = ak.from_series(pd.Series(np.random.choice([True, False], size=10)))
+
+        assert isinstance(p_array, ak.pdarray)
+        assert bool == p_array.dtype
+
+        p_array = ak.from_series(pd.Series([dt.datetime(2016, 1, 1, 0, 0, 1)]))
+
+        assert isinstance(p_array, ak.pdarray)
+        assert np.int64 == p_array.dtype
+
+        p_array = ak.from_series(pd.Series([np.datetime64("2018-01-01")]))
+
+        assert isinstance(p_array, ak.pdarray)
+        assert np.int64 == p_array.dtype
+
+        p_array = ak.from_series(
+            pd.Series(pd.to_datetime(["1/1/2018", np.datetime64("2018-01-01"), dt.datetime(2018, 1, 1)]))
+        )
+
+        assert isinstance(p_array, ak.pdarray)
+        assert np.int64 == p_array.dtype
+
+        with pytest.raises(TypeError):
+            ak.from_series(np.ones(10))
+
+        with pytest.raises(ValueError):
+            ak.from_series(pd.Series(np.random.randint(0, 10, 10), dtype=np.int8))
+
+    @pytest.mark.parametrize("dtype", NUMERIC_SCALARS)
+    @pytest.mark.parametrize("size", pytest.prob_size)
+    def test_fill(self, size, dtype):
+        ones = ak.ones(size)
+        ones.fill(dtype(2))
+        assert (dtype(2) == ones.to_ndarray()).all()
+
+    @pytest.mark.parametrize("size", pytest.prob_size)
+    def test_endian(self, size):
+        a = np.random.randint(1, size, size)
+        aka = ak.array(a)
+        npa = aka.to_ndarray()
+        assert np.allclose(a, npa)
+
+        a = a.newbyteorder().byteswap()
+        aka = ak.array(a)
+        npa = aka.to_ndarray()
+        assert np.allclose(a, npa)
+
+        a = a.newbyteorder().byteswap()
+        aka = ak.array(a)
+        npa = aka.to_ndarray()
+        assert np.allclose(a, npa)
+
+    @pytest.mark.parametrize("size", pytest.prob_size)
+    def test_clobber(self, size):
```

Comment:
since we are converting to and from np arrays, i wouldn't parameterize on `size`

Reply:
Updated